app/experimenter/nimbus-ui/src/components/TableSummary/index.tsx

```diff
@@ -38,7 +39,9 @@ const TableSummary = ({ experiment }: TableSummaryProps) => {
         </tr>
         <tr>
           <th>Hypothesis</th>
-          <td data-testid="experiment-hypothesis">{experiment.hypothesis}</td>
+          <td data-testid="experiment-hypothesis">
+            <RichText text={experiment.hypothesis || ""} />
+          </td>
         </tr>
         <tr>
           <th>Public description</th>
```

Comment:
Does the public description field also need to be `<RichText>` wrapper? I imagine it could potentially include a linebreak and/or hyperlinks.

Reply:
cc @k88hudson @AnaMedinac

Reply:
No, public descriptions have to be rendered in UI for each platform (see about:studies [here](https://searchfox.org/mozilla-central/source/toolkit/components/normandy/content/about-studies/about-studies.js#302)). Keeping it plain text will make it a lot easier to coordinate this across platforms. 

Since this is different from hypothesis, we could add some help text about this being a single paragraph with no formatting or something in help text